### PR TITLE
Update keyboard shortcuts to meet new Style guidelines

### LINKED ISSUE #13878: Update keyboard shortcuts to meet the new GitHub content style guide
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

https://docs.github.com/en/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts

### What part(s) of the article would you like to see updated?

Keyboard shortcuts in this document are outdated. Refer to [Keyboard Shortcuts](https://github.com/github/docs/blob/main/contributing/content-style-guide.md#keyboard-shortcuts) section in the content style guide on how to correctly present keyboard shortcuts in GitHub docs.

### Additional information

<kbd>command</kbd><kbd>k</kbd> should be <kbd>Command</kbd>+<kbd>K</kbd>
<kbd>command</kbd><kbd>shift</kbd><kbd>7</kbd> should be <kbd>Command</kbd>+<kbd>Shift</kbd>+<kbd>7</kbd>

### PR DESCRIPTION
**Pull request by AnilSeervi**
… keys

### Why:

Closes #13878

### Check off the following:

- [ ] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

### CODE REVIEW

File: content/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts.md [hunk 1]
```diff
@@ -16,52 +16,52 @@ GitHub Desktop keyboard shortcuts on macOS
...
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>F</kbd> | Show the repository in Finder
-|<kbd>⇧</kbd><kbd>⌘</kbd><kbd>A</kbd> | Open the repository in your preferred editor tool
-|<kbd>⌘</kbd><kbd>I</kbd> | Create an issue on {% data variables.product.prodname_dotcom %}
+|<kbd>Command</kbd>+<kbd>N</kbd> | Add a new repository
+|<kbd>Command</kbd>+<kbd>O</kbd> | Add a local repository
+|<kbd>Shift</kbd>+<kbd>Command</kbd>+<kbd>O</kbd> | Clone a repository from {% data variables.product.prodname_dotcom %}
+|<kbd>Command</kbd>+<kbd>T</kbd> | Show a list of your repositories
+|<kbd>Command</kbd>+<kbd>P</kbd> | Push the latest commits to {% data variables.product.prodname_dotcom %}
+|<kbd>Shift</kbd>+<kbd>Command</kbd>+<kbd>P</kbd> | Pull down the latest changes from {% data variables.product.prodname_dotcom %}
+|<kbd>Command</kbd>+<kbd>⌫</kbd> | Remove an existing repository
```
Comment (AnilSeervi) [inline content/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts.md:39]: is the icon ok here or should it be `delete`/`backspace`

**Comment by ramyaparimi** [comment]
@AnilSeervi 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts.md (hunk 1 above)
Comment (ramyaparimi) [reply content/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts.md:39]: @AnilSeervi It should be `Delete` as it is spelled as delete on Mac keyboard.

Comment (AnilSeervi) [reply content/desktop/installing-and-configuring-github-desktop/overview/keyboard-shortcuts.md]: Changed it to `Delete`

**Review by ramyaparimi — APPROVED** [review]
@AnilSeervi 
Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

**Review by ramyaparimi — APPROVED** [review]
@AnilSeervi 
Thanks so much for opening a PR with this fix! I'll get this merged once tests are passing 🍏

**Pull request merged**